lib/core/src/lib/auth/oidc/oidc-authentication.service.ts

```diff
@@ -46,11 +46,6 @@ export class OIDCAuthenticationService extends BaseAuthenticationService {
         private readonly auth: AuthService
     ) {
         super(alfrescoApi, appConfig, cookie, logService);
-        this.alfrescoApi.alfrescoApiInitialized.subscribe(() => {
```

Comment:
don't we need it at all?

Reply:
I removed it because the `'logged-in'` event is emitted by the js-api authentication that we want get rid of, but we can't removed it at all, we need to trigger the onLogin because is needed to handle the file version, I'll change that logic using the new auth instead using the alfresco api